exercises/concept/lucas-numbers/README.md

```diff
@@ -0,0 +1,21 @@
+# LucasNumbers
```

Comment:
This looks like it came from running `mix new` and needs to be removed. I usually copy-paste existing exercises instead, to avoid to have to scrap the stuff we don't need every time.

Reply:
Ya I meant to delete this, I'll do that in the next commit